WordPress/WordPressTodayWidget/TodayViewController.swift

```diff
@@ -152,24 +171,51 @@ private extension TodayViewController {
         let data = TodayWidgetStats.loadSavedData()
         visitorCount = data.visitors
         viewCount = data.views
+        likeCount = data.likes
+        commentCount = data.comments
+    }
+
+    func initRows() {
+        guard let row = Bundle.main.loadNibNamed(Constants.rowNibName, owner: nil, options: nil)?.first as? TwoColumnRow else {
```

Comment:
You can use `loadRootViewFromNib` from `Bundle+LoadFromNib`

Reply:
Well, I'd have to include that file in the widget's target. Which I could do. But I don't see a reason to add another file to the widget footprint for just one method when this works fine.

Reply:
Ah got it! Was just to don't have duplication since there's an extension for it, but it's ok if you don't want to include it.